src/libslic3r/PrintConfig.hpp

```diff
@@ -1049,6 +1049,29 @@ PRINT_CONFIG_CLASS_DEFINE(
     ((ConfigOptionBool,                purge_in_prime_tower))
     ((ConfigOptionBool,                enable_filament_ramming))
 
+    // Small Area Infill Flow Compensation
```

Comment:
That's a lot of parameters 🤣

Reply:
> I second this. An array(comma separated strings like the thumbnail size or exclusion_area would be cleaner).

I tried to make that work all day, but whenever I try to use the `coPoints` type in the Tab and have it rendered as a csv string OrcaSlicer instantly crashes before even rendering the UI.

Reply:
Switched to using a multi-line text input instead of the ConfigOptionPoints, that at least doesn't crash the slicer on startup.